posts/2024-08-22_hosting_rshiny_applications.md

```diff
@@ -0,0 +1,33 @@
+---
+media:
+ - mastodon-eu-freiburg
+ - matrix-eu-announce
+ - bluesky-galaxyproject
+ - linkedin-galaxyproject
+
+mentions:
+ mastodon-eu-freiburg:
+  - galaxyproject@mstdn.science
+  - csoneson@fosstodon.org
+  - federicomarini@genomic.social
+  - bgruening@fosstodon.org
+  - hrhotz@genomic.social 
+ bluesky-galaxyproject:
+  - galaxyproject.bsky.social
+ matrix-eu-announce:
+  - bgruening:matrix.org
```

Comment:
That one should be removed :-) it was just for demonstration...

Reply:
removed
(I just used the template 😉 )